variables.tf

```diff
@@ -289,6 +289,20 @@ variable "ssh_keys" {
   }
 }
 
+variable "user_data" {
```

Comment:
We already have a `user_data` field in the `vsi` variable below, so I do not think that we should have a separate variable here.

Is it possible to assign the correct `user_data` to the correct `vsi` object in the pattern instead? That way the override still works and we do not have duplicate variables.